src/app/modules/widgets/tree-viewer/tree-node.ts

```diff
@@ -11,6 +11,7 @@ export interface TreeNode {
   cssClasses?: string;
   hover?: string;
   id?: string;
+  createInContextRequest?: boolean;
```

Comment:
MINOR: how about 'contextualCreateRunning' or 'contextualCreateActive'

Reply:
not needed anymore due to the use of msg bus events